service/src/main/java/bio/terra/externalcreds/controllers/AdminApiController.java

```diff
@@ -21,6 +23,23 @@ public record AdminApiController(
     ExternalCredsConfig externalCredsConfig)
     implements AdminApi {
 
+  @Override
+  public ResponseEntity<Void> putLinkedAccountWithFakeToken(
+      Provider provider, AdminLinkInfo adminLinkInfo) {
+    requireAdmin();
+    var linkedAccount =
+        new LinkedAccount.Builder()
+            .isAuthenticated(true)
+            .provider(provider)
+            .userId(adminLinkInfo.getUserId())
+            .refreshToken("fake-refresh-token")
```

Comment:
I've made it so that trying to get an access token from an eRA Commons account fails with a helpful message instead of some esoteric permissions error.